### ISSUE TITLE
Implement config to have Python classes extend from another base class

### ISSUE THREAD

**Issue by kafkas** [kafkas/typesync#120]
Something like `--custom-pydantic-base=x.y.MyModel` that would produce:

```python
import x.y.MyModel

class TypesyncModel(x.y.MyModel):
    ...

class Example(TypesyncModel):
    ...
```

**Comment by kafkas** [comment]
@mands Just implemented this in #139! You can upgrade to 0.5.0 to use the new `customPydanticBase` option. The behavior is slightly different than what we discussed though. When you provide the `--customPydanticBase=x.y.MyModel` argument, Typesync will parse the `MyModel` class name and `x.y` import path and add the following to the generated file:

```python
from x.y import MyModel
# ...
class TypesyncModel(MyModel):
# ...
```

Hopefully this works for you. Let me know if there are any issues!

**Comment by mands** [comment]
Ah that's great - yes I prefer this implementation actually but thought the other one would be easier to implement. Will be great to try this!
